Open_Science_New

```diff
@@ -384,18 +369,18 @@
 	      <h2><a href="">Hackathons</a></h2>
 	      <tr>
 		<td width="75%" style="vertical-align: middle">
-		  <img src="images/C-BASE.png">
+		  <a href="http://www.digitalinberlin.de/c-base/"><img src="images/C-BASE.png"></a>
 		</td>
 		<td width="25%" style="vertical-align: middle">
 		  <p> Important value of hackathons to data sharing initiatives</p>
-		  <img src="images/Brainhack.png">
+		  <a href="http://www.brainhack.org"><img src="images/Brainhack.png"></a>
 		</td>
 	    </tr></table>
 	  </section>
 	  
 	  <!-- Our implementation -->
 	  <section>
-	    <section data-background="images/mni_eco_info_5.jpg" data-background-size="100%">
+	    <a href="https://samirdas.github.io/MNI_ecosystem.html#/"><img src="images/mni_eco_info_5.jpg" data-background-size="100%"></a>
```

Comment:
Make a note/bookmark -- we will probably want to come back and update this link later, given it points to a presentation that should also get revamped soon.